app/src/test/java/org/simple/clinic/search/results/PatientSearchResultsControllerTest.kt

```diff
@@ -38,196 +28,41 @@ class PatientSearchResultsControllerTest {
   private val uiEvents = PublishSubject.create<UiEvent>()
 
-  private val currentFacility = PatientMocker.facility()
-
   @Before
   fun setUp() {
     RxJavaPlugins.setIoSchedulerHandler { Schedulers.trampoline() }
-
-    val user = PatientMocker.loggedInUser()
-    whenever(userSession.requireLoggedInUser()).thenReturn(Observable.just(user))
-    whenever(facilityRepository.currentFacility(user)).thenReturn(Observable.just(currentFacility))
-    controller = PatientSearchResultsController(
-        patientRepository = patientRepository,
-        userSession = userSession,
-        facilityRepository = facilityRepository,
-        phoneObfuscator = phoneObfuscator,
-        utcClock = utcClock,
-        dateOfBirthFormatter = dateOfBirthFormatter,
-        userClock = userClock
-    )
     uiEvents.compose(controller).subscribe { uiChange -> uiChange(screen) }
   }
 
   @Test
-  fun `when screen is created then patients matching the search query should be shown`() {
-    val patientSearchResult1 = PatientMocker.patientSearchResult()
-    val patientSearchResult2 = PatientMocker.patientSearchResult()
-
-    val patientSearchResults = PatientSearchResults(
-        visitedCurrentFacility = listOf(patientSearchResult1),
-        notVisitedCurrentFacility = listOf(patientSearchResult2)
-    )
-    whenever(patientRepository.search(any())).thenReturn(Observable.just(patientSearchResults))
-
-    uiEvents.onNext(PatientSearchResultsScreenCreated(PatientSearchResultsScreenKey("name")))
-
-    verify(patientRepository).search("name")
-    verify(screen).updateSearchResults(listOf(
-        InCurrentFacilityHeader(facilityName = currentFacility.name),
-        PatientSearchResultRow(
-            searchResult = patientSearchResult1,
-            currentFacility = currentFacility,
-            phoneObfuscator = phoneObfuscator,
-            dateOfBirthFormat = dateOfBirthFormatter,
-            clock = utcClock,
-            userClock = userClock
-        ),
-        NotInCurrentFacilityHeader,
-        PatientSearchResultRow(
-            searchResult = patientSearchResult2,
-            currentFacility = currentFacility,
-            phoneObfuscator = phoneObfuscator,
-            dateOfBirthFormat = dateOfBirthFormatter,
-            clock = utcClock,
-            userClock = userClock
-        )
-    ))
-    verify(screen).setEmptyStateVisible(false)
-  }
-
-  @Test
-  fun `when screen is created and no matching patients are available then the empty state should be shown`() {
-    val emptySearchResults = PatientSearchResults(
-        visitedCurrentFacility = emptyList(),
-        notVisitedCurrentFacility = emptyList()
-    )
-    whenever(patientRepository.search(any())).thenReturn(Observable.just(emptySearchResults))
-
-    uiEvents.onNext(PatientSearchResultsScreenCreated(PatientSearchResultsScreenKey("name")))
+  fun `when SearchResultClicked event is emitted open PatientSummaryScreen`() {
```

Comment:
Rename this to `when search result is clicked, then patient summary must be opened`?

Reply:
done